### PR TITLE
Map javascript file

### PR DESCRIPTION
**Pull request by niccireinking**

### CODE REVIEW

**Review by Rachelware — APPROVED** [review]
lgtm

**Comment by niccireinking** [comment]
I put the function in my html file since I only had one and I couldn't figure out how to work it.  Once I figure it out, I may put the functions in a js file, but for now I just deleted it.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -160,21 +158,36 @@ public User getUser(String email) {
...
+        String idString = newEntity.getKey().getName();
+        UUID id = UUID.fromString(idString);
+        String text = (String) newEntity.getProperty("text");
+        long timestamp = (long) newEntity.getProperty("timestamp");
+        Message message = new Message(id, user, text, timestamp);
+        return message;
+    }
+
+    public void printError(Entity newEntity, Exception e){
+
```
Comment (emto) [inline src/main/java/com/google/codeu/data/Datastore.java:185]: remove whitespace

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -160,21 +158,36 @@ public User getUser(String email) {
...
-                e.printStackTrace();
+                printError(entity, e);
             }
         }
 
         return messages;
     }
+
+    public Message messageQuery(String user, Entity newEntity){
+
```
Comment (emto) [inline src/main/java/com/google/codeu/data/Datastore.java:175]: remove whitespace

it should be like

public void functionName() {
  content
}

without the whitespace before/after

File: src/main/webapp/js/map-loader.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/*
+ * function to create the map
+ */
+function createMap(){
+      const map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: 37.422, lng: -122.084},
```
Comment (emto) [inline src/main/webapp/js/map-loader.js]: add comment for what the lat/long are like:
// Google office MTV

**Review by emto — APPROVED** [review]

**Pull request merged**